wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1006,8 +1006,22 @@ public final class MLSService: MLSServiceInterface {
 
     public func wipeGroup(_ groupID: MLSGroupID) async throws {
         logger.info("wiping group (\(groupID.safeForLoggingDescription))")
+
         do {
-            try await coreCrypto.perform { try await $0.wipeConversation(conversationId: groupID.data) }
+            try await coreCrypto.perform { [self] in
+                guard try await $0.conversationExists(
+                    conversationId: groupID.data
+                ) else {
+                    return logger.info(
+                        "conversation doesn't exist, nothing to wipe.."
+                    )
+                }
+                try await $0.wipeConversation(
+                    conversationId: groupID.data
+                )
+
+                logger.info("wiped group (\(groupID.safeForLoggingDescription))")
```

Comment:
suggestion: move groupID to attributes. (could be not completly exact syntax) i.e:
```suggestion
                logger.info("wiped group", attributes: [.mlsGroupId: (\(groupID.safeForLoggingDescription))])
```

Reply:
done https://github.com/wireapp/wire-ios/pull/2997/commits/84076697b42a00148c2bedbbc998c174aca1a869